src/Instrumentation/PDO/src/PDOInstrumentation.php

```diff
@@ -203,6 +203,7 @@ private static function makeBuilder(
         /** @psalm-suppress ArgumentTypeCoercion */
         return $instrumentation->tracer()
                     ->spanBuilder($name)
+                    ->setAttribute(TraceAttributes::DB_SYSTEM, 'other_sql')
```

Comment:
is https://www.php.net/manual/en/pdo.getattribute.php usable/useful here? Could we map driver names to well-known names from https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#notes-and-well-known-identifiers-for-dbsystem ?